### ISSUE TITLE
Low Contrast and missing ARIA label on auto-populate screen

### ISSUE THREAD

**Issue by jarrod-lowe** [jarrod-lowe/wildsea#856]
- The "auto-populate" section is shown on character sheets when they user may edit the character sheet, and there are no sections on the character sheet
- In both game styles
- There are low contrast warnings on the "Choose a pre-made..." text and the selection box
- The selection box is missing an ARIA label
